### PR TITLE
pdctl: add scheduler describe

### PR DESCRIPTION
**Pull request by CabinfeverB**
Signed-off-by: Cabinfever_B <cabinfeveroier@gmail.com>

### What problem does this PR solve?

Issue Number: ref #5257 
should be merged after #5471

### What is changed and how does it work?

```commit-message
add scheduler describe in pdctl
```

### Check List

Tests

- Unit test 
- Manual test (add detailed scripts or steps below)
- 
image
image

### Release note

```release-note
add scheduler describe in pdctl
```

### CODE REVIEW

File: tools/pd-ctl/pdctl/command/scheduler.go [hunk 1]
```diff
@@ -765,3 +766,40 @@ func setShuffleRegionSchedulerRolesCommandFunc(cmd *cobra.Command, args []string
...
+	return c
+}
+
+func describeSchedulerCommandFunc(cmd *cobra.Command, args []string) {
+	if len(args) != 0 {
+		cmd.Println(cmd.UsageString())
+		return
+	}
+	schedulerName := cmd.Name()
+	url := "pd/api/v1/debug/diagnostic/" + schedulerName
```
Comment (rleungx) [inline tools/pd-ctl/pdctl/command/scheduler.go:797]: make it a constant

**Comment by rleungx** [comment]
Please also add a test.

**Review by disksing — APPROVED** [review]

**Review by lhy1024 — COMMENTED** [review]
would you mind adding an example in pr desc?

File: tools/pd-ctl/pdctl/command/scheduler.go [hunk 2]
```diff
@@ -765,3 +767,40 @@ func setShuffleRegionSchedulerRolesCommandFunc(cmd *cobra.Command, args []string
 	}
 	cmd.Println("Success!")
 }
+
+// NewDescribeSchedulerCommand returns commands to config scheduler.
```
Comment (rleungx) [inline tools/pd-ctl/pdctl/command/scheduler.go:771]:
```suggestion
// NewDescribeSchedulerCommand returns command to describe the scheduler.
```

**Review by nolouch — APPROVED** [review]
overall lgtm

**Pull request merged**